Updates to user research to add additional information (#145)
alphagov/forms: research/2022-10-24_Form_Builder_Usability_Testing.md

> # Form building

Reviewer: ```suggestion
# Usability testing: Form building
```

> ## 2022-10-24 / Sprint 14

Reviewer: ```suggestion
2022-10-24 / Sprint 14
```

## Aims

> To test form builder journeys required to digitise a specific partner department's form. This included testing specific features for creating questions, as well as publishing a live version of the form:

Reviewer: ```suggestion
To test form builder journeys required to digitise a specific partner department’s form. This included testing specific features for creating questions, as well as publishing a live version of the form:
```

- Add a new name to the fom 
- Add a name question 
- Add a address question
- Add content before a question
- Preview a fom

> - Add 'what happens next' page

Reviewer: ```suggestion
- Add ‘what happens next’ page
```

> - Set up emial address for completed form

Reviewer: ```suggestion
- Set up email address for completed form
```

- Add declaration 
- Add a privacy notice

## Users
- Civil Servants
- 6

## Methodology
- Usability testing

## Key Headlines

> - Overall, a very positive experience for all users with many competing the tasks quickly and easily.

Reviewer: ```suggestion
- Overall, a very positive experience for all users with many completing the tasks quickly and easily.
```

# Usability testing: Form building

2022-10-24 / Sprint 14

## Aims
To test form builder journeys required to digitise a specific partner department’s form. This included testing specific features for creating questions, as well as publishing a live version of the form:
- Add a new name to the form 
- Add a name question 
- Add a address question
- Add content before a question
- Preview a form
- Add ‘what happens next’ page
- Set up email address for completed form
- Add declaration 
- Add a privacy notice

## Users
- Civil Servants

> - 6

Reviewer: ```suggestion
- 6 users
```
